### DISCUSSION TITLE
ratroarch

### DISCUSSION THREAD

**Discussion by Shimmer2** [SquirrelJME/SquirrelJME#146] [General Questions] [answered]
When can you play games with the Java me core on the ratroarch?

**Comment by XerTheSquirrel** [comment, marked as answer]
Hi! This is something that is currently being worked on, there currently is no ETA.

**Comment by Shimmer2** [comment]
请问2024年可以完成吗？

**Reply by XerTheSquirrel** [reply]
是的，今年的某个时候。
